lib/mpls.h

```diff
@@ -73,7 +73,7 @@ extern "C" {
 #define MPLS_MAX_LABELS                    16
 
 #define IS_MPLS_RESERVED_LABEL(label)                                          \
-	(label >= MPLS_LABEL_RESERVED_MIN && label <= MPLS_LABEL_RESERVED_MAX)
+	(label > MPLS_LABEL_RESERVED_MIN && label <= MPLS_LABEL_RESERVED_MAX)
```

Comment:
What's the rationale behind this change? I can't see why we shoundn't classify 0 (zero) as a reserved label.

Reply:
I was seeing this warning:

./lib/mpls.h:76:9: warning: comparison of unsigned expression >= 0 is always true [-Wtype-limits]
   76 |  (label >= MPLS_LABEL_RESERVED_MIN && label <= MPLS_LABEL_RESERVED_MAX)
      |         ^~
ldpd/lde.c:2304:9: note: in expansion of macro ‘IS_MPLS_RESERVED_LABEL’
 2304 |     if (IS_MPLS_RESERVED_LABEL(fn->local_label)) {
      |         ^~~~~~~~~~~~~~~~~~~~~~

Reply:
can I just change macro to 

#define IS_MPLS_RESERVED_LABEL(label)                                          \
	(label <= MPLS_LABEL_RESERVED_MAX)

Reply:
yeah that should be enough